modules/browser/BrowserCapabilities.js

```diff
@@ -32,7 +32,6 @@ export default class BrowserCapabilities extends BrowserDetection {
     doesVideoMuteByStreamRemove() {
         return !(
             this.isFirefox()
-            || this.isEdge()
```

Comment:
Since we are here, maybe just say `this.isChromiumBased` instead of negating every other browser.

Reply:
sure

Reply:
sure